### PR TITLE
Hitfall Mode, Airdash Mode, and multi-mode selection for custom game modes

### PR DESCRIPTION
**Pull request by techyCoder81**
- adds a hitfall gamemode next to turbo and tag.
- adds an airdash mode.
- adds the ability to toggle multiple modes at once

### CODE REVIEW

**Comment by techyCoder81** [comment]
it would be cool to also have an airdash mode

File: utils/src/game_modes/airdash.rs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+use smash::phx::Vector3f;
+use crate::util;
+
+pub unsafe fn update() {
+    // skip this frame because the match hasnt started
+    if !app::sv_information::is_ready_go() {
+        return;
+    }
+
+    println!("doing airdash update!");
```
Comment (WuBoytH) [inline utils/src/game_modes/airdash.rs]: We probably shouldn't have these printing every frame if we can help it

**Review by WuBoytH — CHANGES_REQUESTED** [review]

File: utils/src/game_modes/airdash.rs (hunk 1 above)
Comment (techyCoder81) [reply utils/src/game_modes/airdash.rs]: great catch, thanks

**Review by WuBoytH — APPROVED** [review]

**Comment by H3ra-57** [comment]
Works on my end 👍

**Pull request merged**